pytransform3d/rotations/_utils.py

```diff
@@ -59,8 +59,8 @@ def norm_angle(a):
     a_norm : float or array-like, shape (n,)
         Normalized angle(s) in radians
     """
```

Comment:
As the discussion here is very detailed, I would suggest to add a link to it in a comment here.